src/pages/CrossChain/SwapBriefCrossChain.tsx

```diff
@@ -85,7 +79,7 @@ export default function SwapBrief({ route }: Props) {
     }
   }
 
-  const renderAmountUsd = (amountUsdOut: string | undefined) => {
+  const renderAmountUsd = (amountUsdOut: string | undefined | number) => {
```

Comment:
Sorry to be nit-picky, but I like them in a shorter way
```suggestion
  const renderAmountUsd = (amountUsdOut?: string | number) => {
```

Reply:
look different
amountUsdOut: string | undefined | number => force you pass 1 param
amountUsdOut?: string | number => 0 or 1 params

Reply:
Ah okay